lib/features/home/ui/home_screen.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+import '../../../core/helpers/constants.dart';
+import '../../../core/helpers/spacing.dart';
+import '../../../core/theming/styles.dart';
+import 'widget/appbar_home.dart';
+import 'widget/category_slider.dart';
+import 'widget/products.dart';
+import 'widget/search_widget.dart';
+import 'widget/slider_home.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+   
+
+return Scaffold(body:SafeArea(
```

Comment:
please reformat code before push

Reply:
ok I get new MacBook and setting was not good but now every thing good